### ISSUE TITLE
About Test Items acessibility issues

### ISSUE THREAD

**Issue by struthj** [SmarterApp/AP_ItemSampler#4]
http://sampleitems.smarterbalanced.org/AboutItems 

- [ ]  The iframe element used on the page has the following issues: 
After navigating into the iframe, keyboard users cannot exit the iframe using standard keyboard navigation (often referred to as a "keyboard trap"). Ensure that users are able to navigate out of the iframe using Tab (forward) and Shift + Tab (reverse). Severity 1 
Elements in the iframe that are not actionable should not receive keyboard focus (e.g. the numbers, ). Remove tabindex="0" where it is not necessary (see Misuse of tabindex="0"). Severity 3 

~~- [ ] There are dozens of elements that have the lang="enus" attribute applied to them. You should only specify the language inline when there is a change to the language of a specific area when compared to the document language. Remove each of these instances and apply lang="en" to the <html> tag. This will define the document language for the entire page, eliminating the possibility that individual elements may be unintentionally missed. Severity 3~~ 
EDIT I believe this is an iframe issue

- [x] The `<select>` for selecting test item types needs a `<label>`. We recommend an invisible label of "Item type" or similar. 

- [x] On item selection, visual instructions are dynamically changed. Users who are blind will not be aware of this change in content, because both the instruction <div> and the changing-content `<iframe>` have aria-live="polite" attributes, and the `<iframe> `appears to take precedence: 
[aria-live changing content on about items](http://imgur.com/9GcFFLH) 
We recommend removing all aria-live references and letting the user discover the descriptive test while reading through the page. Severity 2 

- [x] The "About This Item" dialog contains several non-interactive elements with tabindex="0". Only the two close "buttons" should receive focus. Remove tabindex="0" from all non-interactive elements. Severity 3 
~~- [ ] The "Zoom out" and "Zoom in" buttons are not keyboard accessible. This is because they have been given `tabindex="-1`", which makes items that normally can receive keyboard focus "unfocusable", meaning you will navigate past them when pressing the Tab key. Remove these attributes and ensure these buttons are accessible with a keyboard. Severity 2~~
EDIT I believe this is an iframe issue

**Comment by struthj** [comment]
I think the select label could be done differently, maybe checking with the the accessibility contractor is a good idea. For now a hidden label was added as suggested in the report

**Comment by struthj** [comment]
Also the Shift+tab problem for Iframe could be done with an event handler, but there may be a better way
